cimgui_funcs.go

```diff
@@ -256,18 +247,22 @@ func (self ImDrawList) CloneOutput() ImDrawList {
 	return (ImDrawList)(unsafe.Pointer(C.DrawList_CloneOutput(self.handle())))
 }
 
-func DrawList_GetClipRectMax(pOut *ImVec2, self ImDrawList) {
+func (self ImDrawList) GetClipRectMax() ImVec2 {
+	pOut := ImVec2{}
```

Comment:
Not really sure... Did i fix it or did i break it?

Reply:
Looks okay to me